util.js

```diff
@@ -39,9 +37,5 @@ function getFsObject (type, file, base) {
   return file
 }
 
-module.exports = {
-  idFromName: idFromName,
-  dirFromName: dirFromName,
-  getDir: getFsObject.bind(null, 'dir'),
-  getFile: getFsObject.bind(null, 'file')
-}
+exports.getDir = (file, base) => getFsObject('dir', file, base)
+exports.getFile = (file, base) => getFsObject('file', file, base)
```

Comment:
I prefer to declare all exports at final, so is more easy to know what is exported.

But could use the object shorthand

```
module.exports = {
  idFromName,
  dirFromName,
  ...
}
```